projects/samples/devices/controllers/camera_recognition/camera_recognition.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 1996-2022 Cyberbotics Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+An example of use of a camera device with recognition capability.
+"""
+
+from controller import Robot
+
+
+class Controller(Robot):
+    def __init__(self):
+        super(Controller, self).__init__()
+        self.timeStep = 64
+        self.camera = self.getDevice('camera')
+        self.camera.enable(self.timeStep)
+        self.camera.recognitionEnable(self.timeStep)
+        self.left_motor = self.getDevice('left wheel motor')
+        self.right_motor = self.getDevice('right wheel motor')
+        self.left_motor.setPosition(float('inf'))
+        self.right_motor.setPosition(float('inf'))
+        self.left_motor.setVelocity(-1.5)
+        self.right_motor.setVelocity(1.5)
+
+    def run(self):
+        while self.step(self.timeStep) != -1:
+            # Get current number of object recognized
+            number_of_objects = self.camera.getRecognitionNumberOfObjects()
+            print(f'Recognized {number_of_objects} objects.')
+            print(' ')
+            objects = self.camera.getRecognitionObjects()
+            counter = 1
+            for object in objects:
+                print(f' Object {counter}/{number_of_objects}: {object.model} (id = {object.id})')
```

Comment:
As we access the attribute directly (instead of user `getModel()` for example), we should update accordingly.